### PR TITLE
Multi-language HMPBs

### PR DESCRIPTION
**Pull request by jonahkagan**
## Overview

Task: #4379 

Adds multi-language HMPB support to VxDesign. As a starting point, supports bilingual ballots for our three translated languages, similar to the BMD ballot.

Since we don't yet have specific customer needs and are just meeting a cert requirement for now, I went with straightforward design decisions as a starting point. We may later learn that we want to prioritize readability in certain ways (e.g. prioritizing one language by styling the other one differently), but we don't know enough to make those decisions today.

## Demo Video or Screenshot

[sample-absentee-ballot-Center_Springfield-1_es-US.pdf](https://github.com/user-attachments/files/15793830/sample-absentee-ballot-Center_Springfield-1_es-US.pdf)
[official-precinct-ballot-North_Springfield-2_zh-Hans.pdf](https://github.com/user-attachments/files/15793833/official-precinct-ballot-North_Springfield-2_zh-Hans.pdf)

## Testing Plan
Added multi-language HMPB fixtures (which are used as visual snapshots and also functionally tested in libs/ballot-interpreter)

## Checklist

- [ ] I have added [logging](https://github.com/votingworks/vxsuite/tree/main/libs/logging) where appropriate to any new user actions, system updates such as file reads or storage writes, or errors introduced.

- [ ] I have added a screenshot and/or video to this PR to demo the change
- [ ] I have added the "user_facing_change" label to this PR to automate an announcement in #machine-product-updates

### CODE REVIEW

File: apps/design/backend/src/language_and_audio/election_strings.ts
Comment (jonahkagan) [inline apps/design/backend/src/language_and_audio/election_strings.ts:1]: Most of this commit is just copying existing patterns

File: libs/hmpb/src/vx_default_ballot_template.tsx [hunk 1]
```diff
@@ -378,7 +535,7 @@ function CandidateContest({
                 top: 0.8,
                 left: -0.9,
                 bottom: 0.2,
-                right: 8.5,
+                right: 8.7,
```
Comment (jonahkagan) [inline libs/hmpb/src/vx_default_ballot_template.tsx:538]: I lowered contest option padding from `0.75rem` to `0.5rem` so this accounts for that by expanding the write-in detection area

File: libs/hmpb/src/base_styles.tsx
Comment (jonahkagan) [inline libs/hmpb/src/base_styles.tsx:1]: Good to hide whitespace for this commit

File: libs/ballot-interpreter/src/interpret_vx_hmpb.test.ts [hunk 2]
```diff
@@ -60,13 +60,13 @@ describe('HMPB - Famous Names', () => {
...
       controller: [],
       mayor: [],
-      'parks-and-recreation-director': [],
       'public-works-director': [],
     });
     assert(backResult.interpretation.type === 'InterpretedHmpbPage');
     expect(backResult.interpretation.votes).toEqual({
       'board-of-alderman': [],
       'city-council': [],
+      'parks-and-recreation-director': [],
```
Comment (jonahkagan) [inline libs/ballot-interpreter/src/interpret_vx_hmpb.test.ts:69]: This contest moved to the second page due to increase in header height

File: apps/design/backend/src/__image_snapshots__/test-decks-test-ts-create-precinct-test-deck-for-a-precinct-with-multiple-ballot-styles-1-snap.png
Comment (jonahkagan) [inline apps/design/backend/src/__image_snapshots__/test-decks-test-ts-create-precinct-test-deck-for-a-precinct-with-multiple-ballot-styles-1-snap.png:1]: there are test code changes after the snapshots - just scroll down

File: libs/hmpb/src/generate_fixtures.test.ts [hunk 3]
```diff
@@ -27,7 +27,10 @@ async function expectToMatchSavedPdf(
     { page: actualPage },
     { page: expectedPage },
   ] of pdfPagePairs) {
-    expect(toImageBuffer(actualPage)).toMatchImage(toImageBuffer(expectedPage));
+    expect(toImageBuffer(actualPage)).toMatchImage(
+      toImageBuffer(expectedPage),
+      { dumpDiffToConsole: true }
+    );
```
Comment (jonahkagan) [inline libs/hmpb/src/generate_fixtures.test.ts:33]: Used this for debugging image diffs, which are otherwise not visible at all, so leaving this in

**Review by kofi-q — APPROVED** [review]
🚢

**Pull request merged**